app/editor/src/components/view/View.tsx

```diff
@@ -0,0 +1,5 @@
+import * as styled from './styled';
+
+export const View: React.FC = ({ children, ...rest }) => {
```

Comment:
This component doesn't work yet.  It should provide a way to have an inline scrollable section so that we don't have pages that lose the header/footer from being displayed.

Reply:
That'll be handy, way easier to track